src/lamplib/src/tests/test_auto_tasks.py

```diff
@@ -430,5 +434,38 @@ def test_patch_tasks(self):
         )
 
 
+def test_generate_all_tasks():
+    """This is a dry run of schedule_global_auto_tasks."""
+    genny_repo_root = os.environ.get("GENNY_REPO_ROOT", None)
+    workspace_root = "."
+    if not genny_repo_root:
+        raise Exception(
+            f"GENNY_REPO_ROOT env var {genny_repo_root} either not set or does not exist. "
+            f"This is set when you run through the 'run-genny' wrapper"
+        )
+    try:
+        reader = YamlReader()
```

Comment:
Running locally I get `Exception: File ./expansions.yml not found.`. Are we able to mock out the expansion data? Or should we put an expansion file fixture somewhere?

Reply:
Ah shoot yeah I thought I was mocking it out already